samples/src/main/java/com/example/bigquery/RunLegacyQuery.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_query_legacy]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableResult;
+
+public class RunLegacyQuery {
+
+  public static void runLegacyQuery() {
+    try {
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests.
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      String query =
+          "SELECT corpus FROM [bigquery-public-data:samples.shakespeare] GROUP BY corpus;";
+
+      QueryJobConfiguration queryConfig =
+          // To use legacy SQL syntax, set useLegacySql to true.
+          QueryJobConfiguration.newBuilder(query).setUseLegacySql(true).build();
+
+      // Execute the query.
+      TableResult result = bigquery.query(queryConfig);
+
+      // Print the results.
+      result
+          .iterateAll()
+          .forEach(row -> row.forEach(val -> System.out.println(val.getStringValue())));
```

Comment:
Better, but this will just print out a bunch of random values on separate lines. Can we preserve the rows together?

```java
System.out.println("Results")
for (FieldValueList row : result.iterateAll()) {
  row.forEach(val -> System.print(val.getStringValue() + " ")
  System.out.println();
}
```

Reply:
The naming was confusing - I will change that.
This now prints out each corpus name and looks like this:
sonnets
various
1kinghenryvi

To preserve the rows, the printout would look like this:
FieldValue{attribute=PRIMITIVE, value=sonnets}
FieldValue{attribute=PRIMITIVE, value=various}
FieldValue{attribute=PRIMITIVE, value=1kinghenryvi}

I think it's better if I just print out the value.